.eslintrc.cjs

```diff
@@ -1,48 +1,12 @@
 module.exports = {
-  extends: [
-    '@mediamonks/eslint-config-base',
-    'plugin:lit/recommended',
-    'plugin:lit-a11y/recommended',
-  ],
   parserOptions: {
     extraFileExtensions: ['.cjs'],
     project: './tsconfig.json',
   },
+  extends: [
```

Comment:
there is no hard rule for this, but every eslint config I have seen in the past had `extends` as the first key in the configuration, could you move it to the top?